src/ui/ui.rs

```diff
@@ -105,12 +105,13 @@ impl UI {
     pub fn init(
         app: &gtk::Application,
         rx: glib::Receiver<Message>,
+        size: Vec<&str>,
         nvim: Rc<RefCell<Neovim>>,
     ) -> Self {
         // Create the main window.
         let window = gtk::ApplicationWindow::new(app);
         window.set_title("Neovim");
-        window.set_default_size(1280, 720);
+        window.set_default_size(size[0].parse().unwrap(), size[1].parse().unwrap());
```

Comment:
Probably should parse things with `size[0].parse().unwrap_or(1280)` and `size[1].parse().unwrap_or(720)` instead, so we don’t crash GNvim if the parse fails (assuming it can fail if the user passes a bad arg; I’m not 100% sure it can). We may want to notify the user too.

Reply:
Sorry, I have to give up here. The critical point is the parsing of the string, and I can't figure out how to do proper try--catch error handling in Rust. A copy&pastable solution would be appreciated.